front/styles/global.css

```diff
@@ -35,3 +35,8 @@ main {
     content: "";
   }
 }
+
+/* Override the default focus styles for input[type="text"] */
+input[type="text"]:focus {
```

Comment:
The problem with this is it will create a differejnce between sparkle and front no ?

Reply:
Yes, this needs to be temporary and removed when we migrate to proper Sparkle Inputs